packages/wonder-blocks-form/src/components/text-field.js

```diff
@@ -7,16 +7,48 @@ import Color from "@khanacademy/wonder-blocks-color";
 import Spacing from "@khanacademy/wonder-blocks-spacing";
 import {styles as typographyStyles} from "@khanacademy/wonder-blocks-typography";
 
-export default class TextField extends React.Component<{||}> {
+type State = {|
+    enabled: boolean,
+    disabled: boolean,
```

Comment:
Based on the previous comment, `disabled` should be treated as a prop that is only defined in the component instance and shouldn't change internally (only via the `disabled` prop).